app/src/main/java/com/sdp/movemeet/view/activity/UploadActivityActivity.java

```diff
@@ -191,6 +191,7 @@ public void handleRegisterUser() {
         if (fAuth.getCurrentUser() != null) {
             userId = fAuth.getCurrentUser().getUid();
             TextView[] textViewArray = {fullName, email, phone};
+            //FirebaseInteraction.retrieveDataFromFirebase(fStore, userId, textViewArray, UploadActivityActivity.this);
             FirebaseInteraction.retrieveDataFromFirebase(fStore, userId, textViewArray, UploadActivityActivity.this);
```

Comment:
Why do you have always the same lines duplicated and the first one commented ?

Reply:
I believe that this code was pulled from your branch, but we can clean the comments up after we have completed the refactoring.

Reply:
You're completely right, I remember now having commented and duplicated those lines ^^'